Source/DafnyRuntime/DafnyRuntimeJava/src/main/java/dafny/DafnySequence.java

```diff
@@ -28,6 +29,14 @@
         return new ArrayDafnySequence<>(elements.clone());
     }
 
+    /**
+     * Return a sequence backed by the given array without making a defensive
+     * copy.  Only safe if the array never changes afterward.
+     */
+    public static <T> DafnySequence<T> unsafeWrapArray(T[] elements) {
```

Comment:
I totally appreciate the need for doing this, but I wonder if we can at least have an `unsafe` boolean on `ArrayDafnySequence` to help debug things if they go sideways.

Reply:
Like, a flag that that just records whether the array was constructed using `unsafeWrapArray`, for debugging purposes?  Sure, that sounds reasonable.